src/tasks/tasks.py

```diff
@@ -412,14 +406,11 @@ def get_sentences(self) -> Iterable[Sequence[str]]:
                 yield sent
 
 
+@register_task('bwb', rel_path='BWB/')
 class WikiTextLMTask(LanguageModelingTask):
```

Comment:
there's something funky going on here...

We might as well merge rename this class (I think the only difference with the base LM class is that we assume a particular format of the raw data) and merge it with the Wiki103 task, maybe even with the base LM class?

Reply:
though maybe this is an issue for a different PR

Reply:
Yeah, I didn't want to do anything that I couldn't easily tell was a no-op.